tools/automation/tests/__init__.py

```diff
@@ -42,6 +102,20 @@ def execute(args):
         output('Running in CI Mode')
         selected_modules = [('All modules', 'azure.cli', 'azure.cli'),
                             ('CLI Linter', 'automation.cli_linter', 'automation.cli_linter')]
+
+        modified_modules = _extract_top_level_modified_modules()
+        if any(base_mod in modified_modules for base_mod in ['core', 'testsdk', 'telemetry']):
+            pass
```

Comment:
>pass [](start = 12, length = 4)

can you add some comments here, like pass means run all test, it took me sometime to understand

Reply:
It's Python list comprehension and will yield a generator as return to any(), expand it to:
```Python
tmp = []
for base_mod in ['core', 'testsdk', 'telemetry']:
    tmp.append(base_mod in modified_modules)

if any(tmp):
    pass
```

Reply:
Comments added